armi/reactor/reactors.py

```diff
@@ -2280,6 +2286,8 @@ def processLoading(self, cs, dbLoad: bool = False):
 
         self.stationaryBlockFlagsList = stationaryBlockFlags
 
+        self.setBlockMassParams()
```

Comment:
So, are we duplicating effort now?

Sorry, I am just looking to see if we are slowing down the code by adding duplicate copies of this same logic. Presumably, these values were already being filled at other time steps. So, do we still need to calculate them at those timesteps in the exact same way?

Reply:
This isn't actually duplicating effort, but filling in these calculations at time steps where they were previously missing. 

But, since you asked, the code is duplicated at those other time steps (at least internally).

Reply:
Until now, our neutronics code for SFRs was calculating and populating `b.p.kgHM` and `b.p.kgFis`. For anyone using a different plugin for neutronics, they would either need to implement a different mass calculation within one of their plugins, or those parameters would just be left as the default value of 0.0.

We have the functions to calculate these parameters in the framework, and it makes more sense to calculate them directly rather than outsource a nearly trivial mass calculation to an external code.

> Presumably, these values were already being filled at other time steps. So, do we still need to calculate them at those timesteps in the exact same way?

At time node 0, when fuel is first loaded into the core, we calculate the heavy metal and fissile mass in each block.
At non-zero time nodes there has been some depletion, so these masses need to be updated. The updated mass calculation should be handled by the depletion interface.